plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorSmokeTest.java

```diff
@@ -160,6 +160,16 @@ public void testWritesLockInvalidContents(String writeStatement, String expected
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testReadingTableWithDeltaColumnInvariant()
+    {
+        assertThat(getQueryRunner().execute("SELECT * FROM invariants").getRowCount()).isEqualTo(1);
+        assertThatThrownBy(() -> computeActual("INSERT INTO invariants VALUES(2)"))
```

Comment:
nit: I think we basically don't use `computeActual()` for DML. I would use `query()` or `assertUpdate()`.